### PR TITLE
added conversion from OpenCV keypoints to LAFs

### PR DESCRIPTION
**Pull request by ducha-aiki**

### CODE REVIEW

File: test/feature/test_laf.py [hunk 1]
```diff
@@ -250,3 +250,30 @@ def test_gradcheck(self):
         assert gradcheck(kornia.feature.extract_patches_from_pyramid,
                          (img, nlaf, PS,),
                          raise_exception=True)
+
+
+class TestConvertFromOpenCVKpts:
+    def test_shape(self):
+        class KP():
+            def __init__(self, x, y, scale, angle):
```
Comment (edgarriba) [inline test/feature/test_laf.py]: No duplication code. Define this class outside or in a separated utils file.

File: kornia/feature/laf.py [hunk 2]
```diff
@@ -366,3 +368,34 @@ def extract_patches_from_pyramid(img: torch.Tensor,
...
+            "kps should be non-empty list of OpenCV keypoints"
+            "Got {}".format(type(kps)))
+    xy = torch.cat([torch.tensor([kp.pt]).view(1, 2, 1) for kp in kps], dim=0)
+
+    # We divide by 2.0, because OpenCV uses diameter, and kornia uses radius.
+    scales = torch.tensor([kp.size * scale_multiplier / 2.0 for kp in kps])
+    # kornia uses counterclockwise convention and starts from x axis
+    angles_deg = torch.tensor([(90 - kp.angle) for kp in kps])
+    rotmat = scales.view(-1, 1, 1) * kornia.angle_to_rotation_matrix(angles_deg)
+    return torch.cat([rotmat, xy], dim=2).unsqueeze(0)
```
[the remark this answers is not in the document]
Comment (edgarriba) [reply kornia/feature/laf.py]: Besides the conversation we had, I also suggest to have separated functions. `convert_opencv_keypoints_to_tensor` which  with a single line you can create a tensor with the keypoint data. An additional `convert_keypoint_to_lafs` that consume tensor and can be computed in batch, and assume that's already in a certain device. This one should have (*,4) [x,y,scale,angle] and return (*,2,3)

File: test/feature/test_laf.py (hunk 1 above)
Comment (lferraz) [reply test/feature/test_laf.py]: why not directly in the module with _?

File: kornia/feature/laf.py [hunk 3]
```diff
@@ -6,6 +6,8 @@
 import torch
 import torch.nn.functional as F
 import warnings
+from typing import List
+
 try:
     import matplotlib.pyplot as plt
```
Comment (lferraz) [inline kornia/feature/laf.py]: what did we decide about matplotlib?

File: kornia/feature/laf.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (lferraz) [reply kornia/feature/laf.py]: I agree with Edgar.
I think it is interesting being able to compute the gradients for the keypoint parameters.

File: kornia/feature/laf.py (hunk 3 above)
Comment (ducha-aiki) [reply kornia/feature/laf.py]: Hmm, looks like the problem with merging "approved" PR - I have done the changes at the time,
but it was not there. Will clean-up with next PR

File: kornia/feature/laf.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ducha-aiki) [reply kornia/feature/laf.py]: How can you compute gradient w.r.t to external data anyway?

[the remark this answers is not in the document]
Comment (lferraz) [reply kornia/feature/laf.py]: that's the point, maybe the data shouldn't be always external. I mean, imagine that you want to develop a keypoint detector and you want to regress the coordinates.

[the remark this answers is not in the document]
Comment (ducha-aiki) [reply kornia/feature/laf.py]: The point of conversion function is to use some external detector, if you need/want to. Develop fully differentiable one - is a separate thing

[the remark this answers is not in the document]
Comment (lferraz) [reply kornia/feature/laf.py]: True, I was assuming that this function could be divided into two functions: the data conversion and the creation of the transformation matrix.
If it is only considered a conversion then it is fine.
